### PR TITLE
Taxonomy doc updates

### PR DESCRIPTION
**Pull request by rsilvery**
Closes #<issue>

### Description Of Changes

- updated data uses & data categories to match fideslang 2.0

### Code Changes

* [ ] _list your code changes here_

### Steps to Confirm

* [ ] _list any manual steps taken to confirm the changes_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Documentation Updated
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: mkdocs/docs/taxonomy/overview.md [hunk 1]
```diff
@@ -31,18 +31,6 @@ Examples of this are:
 
 Learn more about [Data Subjects in the taxonomy reference now](data_subjects.md).
 
-
-### 4. Data Qualifiers
```
Comment (ThomasLaPiana) [inline mkdocs/docs/taxonomy/overview.md:35]: seems like we should have at least something about data qualifiers?

**Review by ThomasLaPiana — APPROVED** [review]

File: mkdocs/docs/taxonomy/overview.md (hunk 1 above)
Comment (rsilvery) [reply mkdocs/docs/taxonomy/overview.md:35]: We deprecated this, correct?

File: mkdocs/docs/taxonomy/data_categories.md [hunk 2]
```diff
@@ -63,52 +63,84 @@ Below is a reference for all subcategories of `system` and `user` to assist with
...
-| `national_identification_number`  | `user.government_id`                      |State issued personal identification number.                                                   |
-| `passport_number`                 | `user.government_id`                      |State issued passport data.                                                                    |
+|  `user.account`  |	   `user`   | 	Account creation or registration information.   | 
+|  `user.authorization`  |	   `user`   | 	Scope of permissions and access to a system.   | 
+|  `user.behavior`  |	   `user`   | 	Behavioral data about the subject.   | 
+|  `user.biometric`  |	   `user`   | 	Encoded characteristics provided by a user.   | 
+|  `user.childrens`  |	   `user`   | 	Data relating to children.   | 
+|  `user.contact`  |	   `user`   | 	Contact data collected about a user.   | 
+|  `user.content`  |	   `user`   | 	Content related to, or created by the subject.   | 
+|  `user.demographic`  |	   `user`   | 	Demographic data about a user.   |
```
Comment (ThomasLaPiana) [inline mkdocs/docs/taxonomy/data_categories.md:73]: I suggest using a markdown formatter, we want to keep this formatting to make it easier to visually parse during editing

File: mkdocs/docs/taxonomy/data_uses.md [hunk 3]
```diff
@@ -79,65 +41,96 @@ The fides key of the organization to which this Data Use belongs.
 
 ## Top Level Data Uses
 
-There are seven top-level Data Use classes:
+The top-level Data Use classes:
```
Comment (ThomasLaPiana) [inline mkdocs/docs/taxonomy/data_uses.md:44]: maybe a nit for another time, but why do we use `classes` here? I know this wasn't part of this change, but it seems like `label` is the term we should be using?

**Pull request merged**